salt/cloud/__init__.py

```diff
@@ -1261,7 +1261,12 @@ def create(self, vm_, local_master=True):
 
                 # a small pause makes the sync work reliably
                 time.sleep(3)
-                client = salt.client.get_local_client(mopts=self.opts)
+
+                mopts_ = salt.config.DEFAULT_MASTER_OPTS
+                mopts_.update(salt.config.minion_config('/etc/salt/master'))
```

Comment:
This will break salt cloud on any system that does not have a hard path ref to /etc/salt/master for the config. Why not use opts['conf_file']?